src/sql/sqlite3/create.sql

```diff
@@ -1,66 +1,80 @@
 create table if not exists "L_ACCOUNTS" (
-"ACCOUNT_ID" integer not null primary key,
-"NAME" varchar(255),
-"BALANCE" float check ("BALANCE" >= "MINIMUM_ALLOWED_BALANCE"),
-"CONNECTOR" varchar(1024),
-"PASSWORD_HASH" varchar(1024),
-"PUBLIC_KEY" text,
-"IS_ADMIN" boolean,
-"IS_DISABLED" boolean,
-"FINGERPRINT" varchar(255),
-"MINIMUM_ALLOWED_BALANCE" float default 0);
+  "ACCOUNT_ID" integer not null primary key,
+  "NAME" varchar(255),
+  "BALANCE" float check ("BALANCE" >= "MINIMUM_ALLOWED_BALANCE"),
+  "CONNECTOR" varchar(1024),
+  "PASSWORD_HASH" varchar(1024),
+  "PUBLIC_KEY" text,
```

Comment:
There's no recommended schema for sqlite because it is not supported for enterprise, so we don't need to work on it.